src/components/Checkout/Checkout.jsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-console */
+import axios from 'axios';
+import React, { useContext, useState } from 'react';
+import { Accordion, AccordionItem, Button, Divider, Input } from '@nextui-org/react';
+import { RadioGroup, Radio } from '@nextui-org/react';
+import { CartContext } from '../Cart/CartContext';
+import { OrderSummary } from './OrderSummary';
+import card from '../../assets/images/card.png';
+import amex from '../../assets/images/american-express.png';
+import paypal from '../../assets/images/paypal.png';
+import visa from '../../assets/images/visa.png';
+import stripe from '../../assets/images/stripe.png';
+
+const initialFormData = {
+  orderItems: [],
+  paymentMethod: 'Credit Card',
+  shippingAddress: {
+    address: '',
+    city: '',
+    state: '',
+    postcode: '',
+    phone: ''
+  }
+};
+
+const confirmOrder = async (formData) => {
+  try {
+    const response = await axios.post('http://localhost:500/api/order', formData);
```

Comment:
Please use `api` from `src/configs/api.js` instead to fetch the data, it's already configured to work with multiple enviroments.